OxfordBodleian/Juel-Jensen/BDLaethb4.xml

```diff
@@ -183,63 +1264,198 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    different tools. The blind tooled motif, a cross surrounded by
+                                    several frames, is identical on the front and the back board. </decoNote>
+                                <decoNote xml:id="b4" type="SewingStations">4</decoNote>
+                                <decoNote xml:id="b5" type="Endbands" color="brown">
+                                    <term key="slitBraid">slit-braid</term> endbands were sewn to
+                                    the text block of which only the headband survived partly.
+                                </decoNote>
                             </binding>
-                        </bindingDesc>-->
-                        
+                        </bindingDesc>
+
                     </physDesc>
-                    
-                <!--    <history>
+
+                    <history>
                         <origin>
-                            <origDate when="" evidence=""/>
-                        </origin>
-                        <provenance>  </provenance>
-                    </history>-->
-                    
+                            <origDate from="1600" to="1799" evidence="lettering"> </origDate> The
+                            earliest date of production – taken the handwriting into account – can
+                            be set to the 17th century. The name of <persName ref="PRS13193Giyorgis"
+                                >Giyorgis</persName> is mentioned in the supplication formulas
+                            throughout the manuscript following the phrase <foreign xml:lang="gez"
+                                >ገበርከ</foreign> ‘your servant’. This person was either the patron,
+                            the owner, the donor or the scribe of the manuscript. Names of members
+                            of his family are often stated along with him: his wife <persName
+                                ref="PRS13194AmataSell">ʾAmata Śǝllāse</persName> and his children
+                                <persName ref="PRS13195GadlaGiyorgis">Gadla Giyorgis</persName>,
+                                <persName ref="PRS13196WaldaMik">Walda Mikāʾel</persName> and
+                                <persName ref="PRS13197WalattaMik">Walatta Mikāʾel
+                            </persName>.</origin>
+
+                        <provenance> The manuscript must have come into the possession of the church
+                            of Maqdala Madḫāne ʾĀlam <ref target="INS0101MadhaneAlam"/> at a certain
+                            time in its history from where it was proably looted on 13 April 1868
+                            (see <ref target="#a1"/>) and brought to England. It is unknown how and
+                            when the manuscript came into the possession of <persName
+                                ref="PRS5782JuelJen"/>. to be checked: Bought by the </provenance>
```

Comment:
`to be checked: Bought by the` ?

Reply:
Yes, I somehow partly deleted my comment, a reminder to check one more source for studying the history of the ms.